packages/notifier/src/publish-kit.js

```diff
@@ -188,6 +136,17 @@ export const makePublishKit = () => {
         );
       }
     },
+    getUpdateSince: updateCount => {
+      if (updateCount === undefined) {
+        return subscriber.subscribeAfter().then(makeUpdateRecord);
+      }
+      updateCount = BigInt(updateCount);
+      // We ensure we're at a fresh publication record in case they came in a
+      // big batch.
+      return subscriber
+        .subscribeAfter(updateCount)
+        .then(() => subscriber.getUpdateSince());
```

Comment:
I don't understand this comment, can you elaborate on the kind of scenario you want to affect?

Reply:
Is this clearer?